README.md

```diff
@@ -20,10 +20,10 @@ v0.12.0 was the last version of MapboxStatic to support the [Legacy Static Image
    * Swift Package Manager 5.3 or above
 * Xcode 11 or above (Xcode 12 or above if using Swift Package Manager)
```

Comment:
The CircleCI configuration only builds the library in Xcode 12, so we should stop claiming it supports Xcode 11.